LibreNMS/Alert/Transport/Hipchat.php

```diff
@@ -50,9 +50,12 @@ public function contactHipchat($obj, $option)
         if ($version == 2) {
             $url .= '/' . urlencode($option['room_id']) . '/notification?auth_token=' . urlencode($option['auth_token']);
         }
+        /*
         foreach ($obj as $key => $value) {
```

Comment:
??

Reply:
Should be deleted. $method doesn't exist. $api is not used